libraries/AP_Frsky_Telem/AP_Frsky_Telem.h

```diff
@@ -127,7 +127,7 @@ class AP_Frsky_Telem
     void update_control_mode(uint8_t mode) { _ap.control_mode = mode; }
 
     // update whether we're flying (used for Plane)
-    void set_is_flying(bool is_flying) { *_ap.value == is_flying ? (*_ap.value | AP_ISFLYING_FLAG) : (*_ap.value & (~AP_ISFLYING_FLAG)); }
+    void set_is_flying(bool is_flying) { _ap.value == is_flying ? (_ap.value | AP_ISFLYING_FLAG) : (_ap.value & (~AP_ISFLYING_FLAG)); }
```

Comment:
good catch. The logic should not be ==, simply checking is_flying is what you want.